### PR TITLE
Support Lotus

### PR DESCRIPTION
**Pull request by Nerian**
Hi,

This pull requests makes bootstrap-sass compatible with the Lotus framework.

https://github.com/lotus/assets

### CODE REVIEW

File: lib/bootstrap-sass.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ def load!
 
       if rails?
         register_rails_engine
+      elsif lotus?
+        Lotus::Assets.sources << assets_path
```
Comment (cvrebert) [inline lib/bootstrap-sass.rb]: This should probably go inside a `register_lotus` method.

Comment (Nerian) [inline lib/bootstrap-sass.rb]: Fixed

**Comment by glebm** [comment]
Thanks! You might also want to add this to https://github.com/twbs/bootstrap-rubygem

**Comment by Nerian** [comment]
@glebm Sure, will do :) What's the difference between that project and this one?

**Comment by glebm** [comment]
@Nerian This one is for Bootstrap 2 and 3, the other one is for the upcoming Bootstrap 4 currently in alpha.

**Comment by Nerian** [comment]
I see :)

Pull request done: https://github.com/twbs/bootstrap-rubygem/pull/17

**Pull request merged**